README.md

```diff
@@ -13,9 +13,12 @@ To run with the defaults:
 
 ```bash
 docker run --rm --name frontier-squid \
-  -p 3128:3128 opensciencegrid/osg-frontier-squid:development
+  -p 3128:3128 opensciencegrid/osg-frontier-squid:stable
```

Comment:
```suggestion
  -p 3128:3128 opensciencegrid/osg-frontier-squid:fresh
```
to match the Dockerfile?

Reply:
I think it makes sense to tell users to recommend installing from the stable tag analogous to how we tell users to install out of osg-release instead of osg-testing

Reply:
Uh this PR is about renaming "development" to "fresh", not changing it to stable. That's a separate issue.

Reply:
If you want me to make that change in another PR, I can do that. But ultimately I think we should default to telling our users to install `stable` images

Reply:
A separate PR or at least a separate commit, please.

Reply:
Fixed